src/styles/blocks/card.scss

```
@@ -0,0 +1,77 @@
+.card{
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $bg-color;
+  padding: 32px 16px 16px;
+  transition:  0.3s transform;
+  margin: 0 auto;
+
+
+  &:hover{
+    transform: scale(1.2);
```

fix this space in all cases
```suggestion
  &:hover {
    transform: scale(1.2);
```